app/controllers/message_threads/bulk/message_drafts_controller.rb

```diff
@@ -28,7 +28,7 @@ def destroy
         message_drafts_to_destroy_ids = message_threads.map(&:message_drafts).flatten.map(&:id)
 
         message_threads.transaction do
-          MessageDraft.where(id: message_drafts_to_destroy_ids).destroy_all
+          MessageDraft.not_in_submission_process.where(id: message_drafts_to_destroy_ids).destroy_all
```

Comment:
@luciajanikova nie je toto nejako zavadzajuce, ze oznacis nieco na odoslanie a ono sa to neodosle vlastne (lebo to uz je zaradene na odoslanie). Ci to je by design takto?

Reply:
Nerozumiem asi uplne otazke, co presne myslis. Vies rozvinut @jsuchal?

Reply:
Nezdá sa mi že niečo označíš že chces zmazat a potichu to nepôjde ale asi to dáva zmysel. Iné riešenie by bolo zložité (zisťovať či môžeš zmazať a potom upozorniť...)